akka-http-core/src/test/scala/akka/http/scaladsl/model/EntityDiscardingSpec.scala

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.scaladsl.model
+
+import akka.Done
+import akka.http.scaladsl.model.HttpEntity.Chunked
+import akka.http.scaladsl.{ Http, TestUtils }
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl._
+import akka.testkit.AkkaSpec
+import scala.concurrent.duration._
+import akka.util.ByteString
+
+import scala.concurrent.{ Await, Promise }
+
+class EntityDiscardingSpec extends AkkaSpec {
```

Comment:
FYI @krzemin - I missed this during review (previous test leaked the ActorSystem), AkkaSpec spins up and takes care to shut down the ActorSystem, if an AS is needed use this instead :)